frame/contracts/src/wasm/runtime.rs

```diff
@@ -508,7 +504,7 @@ impl<'a, E: Ext + 'a> Runtime<'a, E> {
 					},
 					Termination =>
 						Ok(ExecReturnValue { flags: ReturnFlags::empty(), data: Vec::new() }),
-					SupervisorError(error) => return Err(error.into()),
+					SupervisorError(error) => return Err(error.clone().into()),
```

Comment:
This `clone` needed?

Reply:
No this is just a leftover from when I was using `downcast_ref`. Clippy is already complaining :)